src/data_enrichment/data_enrichment.py

```diff
@@ -0,0 +1,48 @@
+from translator import Translator
+from paraphraser import Paraphraser
+
+class DataEnrichment:
+    def __init__(self):
+        self.translator = Translator()
+        self.paraphraser = Paraphraser()
+
+    def enrich_data(self, text, lang):
```

Comment:
add parameter types and return types

Reply:
In every function

Reply:
updated all the functions with parameter types and return info